src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -116,7 +116,12 @@ const FiltersWithScrollIntoView: React.FC<{
   const scrollRef = useRef<HTMLDivElement | null>(null)
 
   return (
-    <Box ref={scrollRef as any} overflowY="scroll" height="100%">
+    <Box
+      ref={scrollRef as any}
+      overflowY="scroll"
+      height="100%"
+      data-testid="FiltersWithScrollIntoView"
```

Comment:
was impossible to query it from the tests with RTL without the testid, it's the only non-dev change